client/src/main/java/sigblockchain/projectred/client/Block.java

```diff
@@ -0,0 +1,82 @@
+package sigblockchain.projectred.client;
+
+public class Block {
+    private final int Version;          // uint16
```

Comment:
all instance variables should start with lowercase. This is a Java naming convention. https://www.oracle.com/technetwork/java/codeconventions-135099.html

Reply:
Fixed!